### DISCUSSION TITLE
Don't install develop branch for Jellyseerr as default

### DISCUSSION THREAD

**Discussion by pafade89** [tteck/Proxmox#3191] [Q&A] [answered]
Is it possible to add a selection option to the Jellyseerr script to choose which branch to install? Currently, it always installs the develop branch, but it might make more sense to default to the stable one.

**Comment by tteck** [comment, marked as answer]
I will change that when I find time.

**Reply by pafade89** [reply]
First of all, thank you for your incredibly quick response time and secondly for the excellent support (this isn't the first time 😉).

**Reply by pafade89** [reply]
Thx, now it is only beta
